### PR TITLE
document `tidb_load_based_replica_read_threshold`

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

Add a section to describe the scenario of `tidb_load_based_replica_read_threshold`.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/13584
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by Oreoxmt** [comment]
@you06 @hfxsd PTAL

**Review by hfxsd — APPROVED** [review]

File: system-variables.md [hunk 1]
```diff
@@ -2559,16 +2559,27 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
 ### tidb_load_based_replica_read_threshold <span class="version-mark">New in v7.0.0</span>
 
-> **Warning:**
->
-> The feature controlled by this variable is not fully functional in the current TiDB version. Do not change the default value.
+<CustomContent platform="tidb">
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Default value: `"0s"`
```
Comment (you06) [inline system-variables.md:2566]:
```suggestion
- Default value: `"1s"`
```

The default value has changed yesterday.

File: system-variables.md [hunk 2]
```diff
@@ -2559,16 +2559,27 @@ For a system upgraded to v5.0 from an earlier version, if you have not modified
...
+- Type: String
+- This variable is used to set the threshold for triggering load-based replica read. When the estimated queue time of the leader node exceeds the threshold, TiDB prioritizes reading data from the follower node. The format is a time duration, such as `"100ms"` or `"1s"`. For more details, see [Troubleshoot Hotspot Issues](/troubleshoot-hot-spot-issues.md#scatter-read-hotspots).
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
 
 - Scope: SESSION | GLOBAL
 - Persists to cluster: Yes
 - Default value: `"0s"`
```
Comment (you06) [inline system-variables.md:2577]:
```suggestion
- Default value: `"1s"`
```

Ditto.

**Comment by Oreoxmt** [comment]
@you06 PTAL again

**Review by you06 — APPROVED** [review]

**Pull request merged**